docker/elasticsearch-setup/build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
```

Comment:
Question:
Why do `elasticsearch`, `kafka` & `mysql` setup jobs require build.gradle files but Postgres does not?

Reply:
Only considering quickstart related components, postgres is not used by quickstart. An alternative like quickstartPg or something could be added in the future.